pkg/pretty/context_builder.go

```diff
@@ -18,26 +18,59 @@ package pretty
 
 import (
 	"fmt"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // ContextBuilder allows building up pretty message lines with context
 // that is important for debugging and tracing. This class helps create log
 // line formatting consistency. Pretty lines should be in the form:
 // <Kind> "<Namespace>/<Name>": <message>
```

Comment:
Need to update comment form with new optional version:  `<Kind> "<Namespace>/<Name> v<Version>": <message>`

Reply:
Thanks. Fixed it.

Reply:
Is it me or did you not update this one on line 29?  There was a second one I think but it looks like it was addressed...

Reply:
Yeah, I missed this one. There was one similar further down. Both are now fixed.